### PR TITLE
docs: overhaul community page

### PR DESCRIPTION
**Pull request by Jcahilltorre**
Updates the community page to split the content into sections.

Removes the link to slack and prioritizes GitHub issues as means of communication.

### CODE REVIEW

File: source/community.rst [hunk 1]
```diff
@@ -17,41 +21,70 @@ Recorded sessions:
...
+*************************
+Reporting bugs and issues
+*************************
 
-************
-Contribution
-************
+You can report bugs and issues on GitHub:
 
+<<<<<<< HEAD
```
Comment (avahahn) [inline source/community.rst:30]: Was this a merge conflict you meant to resolve?

File: source/community.rst [hunk 2]
```diff
@@ -17,41 +21,58 @@ Recorded sessions:
...
-Command-line tools:
+***************************
+Reporting security concerns
+***************************
 
-- GitHub repository: https://github.com/nginx/unit/tree/master/tools
+Please report any security-related issues concerning Unit to
+`security-alert@nginx.org <security-alert@nginx.org>`__.
+For our mutual convenience, specifically mention **NGINX Unit** in the subject and follow the
+`CVSS v3.1 <https://www.first.org/cvss/v3.1/specification-document>`__.
```
Comment (callahad) [inline source/community.rst:46]: Small grammar nit with the removal of "specification"

```suggestion
For our mutual convenience, specifically mention **NGINX Unit** in the subject and follow
`CVSS v3.1 <https://www.first.org/cvss/v3.1/specification-document>`__.
```

Comment (javorszky) [inline source/community.rst:46]: this one reads as

> and follow the CVSS v3.1

Is that intended? I'm half expecting another word, like so:

> and follow the CVSS v3.1 specification

Not a blocker.

**Review by javorszky — APPROVED** [review]
One non-blocker question, otherwise looks good!

File: source/community.rst [hunk 3]
```diff
@@ -17,41 +21,58 @@ Recorded sessions:
...
 
-Documentation:
+************
+Contributing
+************
 
-- GitHub repository: https://github.com/nginx/unit-docs
+We welcome contributions to the NGINX Unit project.
+If you have a feature request or a bug fix, please submit a pull request on the
+`unit repo <https://github.com/nginx/unit-docs/pulls>`_.
```
[the remark this answers is not in the document]
Comment (Jcahilltorre) [reply source/community.rst:54]: Kinda but not? 

The section above is specific about bug reports.

Here we are also encouraging users to contribute with new features. I will reword it.

File: source/community.rst [hunk 4]
```diff
@@ -17,41 +21,58 @@ Recorded sessions:
...
 
-Source code:
+- If you have found a fix or want to suggest a change to the code,
+  you can open a `pull request <https://github.com/nginx/unit-docs/pulls>`_
+  on the Unit repo.
 
-- GitHub repository: https://github.com/nginx/unit
+You can also ask questions and discuss your suggestions for Unit by
+`subscribing <https://mailman.nginx.org/mailman3/lists/unit.nginx.org/>`_
+to the unit@nginx.org mailing list and posting your ideas there.
```
[the remark this answers is not in the document]
Comment (ac000) [reply source/community.rst:38]: I'd like to at least retain a mention of the mailing list.

File: source/community.rst (hunk 2 above)
Comment (Jcahilltorre) [reply source/community.rst]: reworded the sentence based on dan's feedback

File: source/community.rst [hunk 5]
```diff
@@ -17,41 +21,62 @@ Recorded sessions:
...
+before submitting your pull request.
 
-Source code:
+You can also ask questions and discuss your suggestions for Unit:
 
-- GitHub repository: https://github.com/nginx/unit
+- Visiting the `Discussions section <https://github.com/nginx/unit/discussions>`_
+  of the Unit repo.
+- By `subscribing <https://mailman.nginx.org/mailman3/lists/unit.nginx.org/>`_
+  to the unit@nginx.org mailing list and posting your ideas there.
```
Comment (callahad) [inline source/community.rst:61]:
```suggestion
- Visiting the `Discussions section <https://github.com/nginx/unit/discussions>`_
  of the Unit repo.
- Subscribing to the `unit@nginx.org mailing list <https://mailman.nginx.org/mailman3/lists/unit.nginx.org/>`_
  and posting your ideas there.
```

File: source/community.rst [hunk 6]
```diff
@@ -17,41 +21,62 @@ Recorded sessions:
...
-to the unit@nginx.org mailing list
-and post your ideas there.
-Also, you can open a pull request on GitHub
-or join our
-`Slack community <https://community.nginx.org/joinslack>`__.
+Please review the `Contributing Guidelines <https://github.com/nginx/unit/blob/master/CONTRIBUTING.md>`_
+before submitting your pull request.
 
-Source code:
+You can also ask questions and discuss your suggestions for Unit:
```
Comment (callahad) [inline source/community.rst:56]:
```suggestion
You can also ask questions and discuss your suggestions for Unit by:
```

**Review by callahad — APPROVED** [review]
LGTM. One small wording tweak for your consideration included in comments. Take it or leave it :)

**Pull request merged**